types/node/http2.d.ts

```diff
@@ -1053,7 +1053,6 @@ declare module 'http2' {
          */
         unknownProtocolTimeout?: number | undefined;
         selectPadding?(frameLen: number, maxFrameLen: number): number;
-        createConnection?(authority: url.URL, option: SessionOptions): stream.Duplex;
```

Comment:
`types/node/ts4.8/http2.d.ts` also.

 Make sense to fix for `v14`/`v16`/`v14/ts4.8`/`v16/ts4.8` too.